### DISCUSSION TITLE
Remove widnows names and text on Toolbars

### DISCUSSION THREAD

**Discussion by claudiofranck** [valinet/ExplorerPatcher#731] [Ideas]
If there is  was to remove or disable the text with the names for the programs on the toolbar please let me know.
If not adding that again could be nice.

**Comment by chip33** [comment]
Hi there ,
Here is the solution you are looking for if you mean Taskbar not Toolbar. :)
(https://github.com/valinet/ExplorerPatcher/discussions/721)

**Comment by claudiofranck** [comment]
Hello, sorry I di not make myself clear, i really meant toolbar but the toolbar on the taskbar, the row on top 
![Screenshot 2022-01-26 145644](https://user-images.githubusercontent.com/98473392/151237551-6b31d4fb-7d57-4924-9de5-081f5900c49c.png)

you get to it right clicking on the taskbar and selecting toolbars and select the one you want or you can create selecting a folder with your shortcuts.

**Reply by matzoshaina** [reply]
Unlock the taskbar. Right click on the dividers between each toolbar. In the right click menu unclick show text.

**Comment by claudiofranck** [comment]
Thanks a lot that did it
